rita_common/src/network_endpoints/mod.rs

```diff
@@ -19,13 +21,13 @@ pub struct JsonStatusResponse {
 impl JsonStatusResponse {
```

Comment:
are we sure we still need this? I think this sort of conversion is done automatically by actix web now.

Reply:
fixed